FactoryMethodTask/Program.cs

```diff
@@ -1,13 +1,17 @@
 ﻿using System;
+using FactoryMethodTask.shapes;
+using FactoryMethodTask.shapes.bases;
 
-namespace FactoryMethodtask
+namespace FactoryMethodTask
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            IShape shape = ShapeFactory.GetRandomShape();
-
+            ClientShape client = new ClientShape();
+            
+            IShape shape = client.GetRandomShape();
+       
```

Comment:
You should move this logic into the GetRandomShape() method.
Also it would be nice if you implement a function that will prompt to user in console asking for parameters for creating a shape, e.g. width, height in case of Rectangle; radius -- in case of Circle; side length of Triangle.
For instance: "Please enter height:", then "Please enter width:" etc.

Reply:
Ok I got it, I will do it